skbio/metadata/_interval.py

```diff
@@ -90,6 +91,10 @@ def __init__(self, interval_metadata, locations,
 
     def _add(self):
         """Add the current ``Interval`` to the IntervalMetadata object."""
+        if self.locations[-1][-1] > self._interval_metadata.upper_bound:
```

Comment:
It would be nice to have a read-only `lower_bound` property that always returns zero, just for symmetry with the `upper_bound` property. Users wouldn't be able to configure this but it could be possible in the future.